creator-node/src/routes/healthCheck.js

```diff
@@ -90,6 +90,10 @@ module.exports = function (app) {
   }))
 
   app.get('/version', handleResponse(async (req, res) => {
+    if (config.get('isReadOnlyMode') === true) {
```

Comment:
nit -- don't need to check `=== true`, just `if (config.get('isReadOnlyMode')`

Reply:
Fixed in https://github.com/AudiusProject/audius-protocol/pull/1183, thanks!